rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -1954,6 +1954,26 @@ public String getVersion() {
         }
     }
 
+    @Override
+    public boolean existsFileInGlobalSpace(String sessionId, String pathname) throws RestException {
+        SchedulerProxyUserInterface proxy = checkAccess(sessionId);
```

Comment:
Should  be this line part of the try-catch block ?

Reply:
checkAccess only throws NotConnectedRestException which is not need to be handled (expected to be directly thrown)